openquakeplatform/openquakeplatform/static/css/risk_input_model.css

```diff
@@ -0,0 +1,103 @@
+
+
+#retrofittingSelect {
+	margin-left: 30px
+}
+
+label {
+	width: 30%;
+	text-align: right;
+	padding-right: 2%;
+	margin-left: 5px;
+}
+
+.ffsForm {
+	float: left;
+	margin-top: 20px;
+}
+
+.vfsForm {
```

Comment:
(like in http://stackoverflow.com/questions/2099252/2-classes-with-different-name-should-have-same-property-in-css-without-repeating)